Makefile

```
@@ -453,8 +453,8 @@ ifneq ($(wildcard $(MAGMA_DIR)/lib/libmagma.*),)
   BACKENDS_MAKE += $(MAGMA_BACKENDS)
 endif
 
-BACKENDS_ENV ?= BACKENDS
-ifneq ($(BACKENDS_ENV),)
+BACKENDS_ENV = $(BACKENDS)
+ifeq ($(BACKENDS_ENV),)
   BACKENDS = $(BACKENDS_MAKE)
   export BACKENDS
 endif
```

What semantic are you looking for in which this isn't sufficient?
```suggestion
BACKENDS ?= $(BACKENDS_MAKE)
```

---

It's really just an artifact of getting the old broken code to work. I think your change would be preferable.

---

Woops, we still need to export ttis variable for CI... We should meld the two